app/components/candidate_interface/complete_section_component.html.erb

```diff
@@ -1,20 +1,8 @@
-<%= form_with model: section_complete_form, url: path, method: request_method do |f| %>
-  <%= f.govuk_error_summary %>
-
-  <%= render(review_component) %>
-
-  <%= f.govuk_collection_radio_buttons :completed,
-    [[true, complete_or_reviewed_radio_button_label], [false, t('application_form.incomplete_radio')]],
-    :first,
-    :last,
-    legend: { text: t('application_form.completed_question'), size: 'm' },
-    hint: -> do %>
-      <% if hint_text %>
-        <div class="govuk-!-width-two-thirds">
-          <%= hint_text %>
-        </div>
-      <% end %>
-  <% end %>
-
-  <%= f.govuk_submit t('continue') %>
-<% end %>
+<div class="govuk-!-width-two-thirds">
```

Comment:
Passing the custom styling for the hint as a Proc now caused a rendering issue and I have no idea why 😱  For now I have reverted back to wrapping the radio buttons in a div.

<img width="827" alt="hnit_proc_madness" src="https://user-images.githubusercontent.com/5256922/116697779-8200c880-a9bb-11eb-9a3e-565f610ffb52.png">

Reply:
Hmm, not sure. What happens if you move the hint text to a helper and call it like this? If it's the same I'll take a closer look.

```erb
  <%= form.govuk_collection_radio_buttons :completed,
      [[true, complete_or_reviewed_radio_button_label], [false, t('application_form.incomplete_radio')]],
      :first,
      :last,
      legend: { text: t('application_form.completed_question'), size: 'm' },
      hint: -> { some_hint_text_helper } %>
```

Reply:
### Does not work

```

```

### Almost works

```
 <%= form.govuk_collection_radio_buttons :completed,
      [[true, complete_or_reviewed_radio_button_label], [false, t('application_form.incomplete_radio')]],
      :first,
      :last,
      legend: { text: t('application_form.completed_question'), size: 'm' },
      hint: -> do %>
        <%= "<div class=\"govuk-!-width-two-thirds govuk-hint\">#{hint_text}</div>".html_safe %>
  <% end %>
```

<img width="560" alt="almost_works" src="https://user-images.githubusercontent.com/5256922/116702929-93e56a00-a9c1-11eb-9c17-1752d06a77c4.png">


> > As to what happens when you apply again, given it’s a different question (or at least is presented as one), I agree with @frankieroberto that it should revert to being unspecified as you would have answered that specific question before.
> 
> Ha, I thought I'd suggested the opposite, that the "review" sections would display this, rather than having neither selected:
> 
> > **Have you reviewed this section?**
> > ( ) Yes, I have reviewed this section
> > (*) No, I'll come back to it later
> 
> – but I don't feel too strongly on this.

@paulrobertlloyd @frankieroberto let's re-group on Tuesday to discuss and confirm requirements 👍

Reply:
Odd, locally this worked for me. Without the `safe_join` it outputs an empty div, maybe try `text.html_safe`?

### `_form.html.erb`

```erb
  <%= form_with(model: order) do |form| %>
  <%= form.govuk_error_summary %>

  <%= form.govuk_radio_buttons_fieldset(
    :extra_topping_id,
    legend: { text: "Select your toppings" },
    hint: -> { topping_hint }) do
  %>

    <%= form.govuk_radio_button(:extra_topping_id, "ham", label: { text: "Ham" }) %>
    <%= form.govuk_radio_button(:extra_topping_id, "pineapple", label: { text: "Pineapple 🍍" }) %>
    <%= form.govuk_radio_button(:extra_topping_id, "sweetcorn", label: { text: "Sweetcorn" }) %>
    <%= form.govuk_radio_button(:extra_topping_id, "caviar", label: { text: "Caviar" }) %>
    <%= form.govuk_radio_button(:extra_topping_id, "saffron", label: { text: "Saffron" }) %>
  <% end %>

  <%= form.govuk_submit("Place order") %>
<% end %>                                                                                     
```

### `orders_helper.rb`

```ruby
module OrdersHelper
  def topping_hint
    tag.div do
      safe_join(
        [
          tag.p("You will be charged extra for"),
          tag.ul do
            safe_join(
              [
                tag.li("Caviar"),
                tag.li("Saffron"),
              ],
            )
          end,
        ],
      )
    end
  end
end
```

### Result

![Screenshot from 2021-05-04 15-25-35](https://user-images.githubusercontent.com/128088/117019109-053c5a00-aced-11eb-8b0f-282162b072b0.png)